pages/index.md

```diff
@@ -19,20 +19,10 @@ To install a nightly build of PnP PowerShell:
 Install-Module -Name PnP.PowerShell -AllowPrerelease -SkipPublisherCheck -AllowClobber
 ```
 
-To use PnP.PowerShell in a Windows container:
-
-```
-docker run -it m365pnp/powershell:1.10.0-nanoserver-1809
-```
-
-To use PnP.PowerShell in a Linux container:
-
-```
-docker run -it m365pnp/powershell
-```
-
 See the [articles](/powershell/articles) section for more information on authentication and configuration. All [cmdlets](/powershell/cmdlets/Add-PnPAlert.html) have been documented too.
 
+See [Using PnP PowerShell in Docker containers](/powershell/articles/docker.html) article for information how to run PnP.PowerShell without installing it.
```

Comment:
Can we please keep the above code snippet for user s who know Docker and also add this description below that ?